pkg/server/domain/service/user.go

```diff
@@ -66,32 +62,40 @@ func NewUserService() UserService {
 	return &userServiceImpl{}
 }
 
-func (u *userServiceImpl) Init(ctx context.Context) error {
+// AdminConfigured check if admin user is initialized
+func (u *userServiceImpl) AdminConfigured(ctx context.Context) (apisv1.AdminConfiguredResponse, error) {
 	admin := model.DefaultAdminUserName
-	if err := u.Store.Get(ctx, &model.User{
+	adminUser := &model.User{
 		Name: admin,
-	}); err != nil {
-		if errors.Is(err, datastore.ErrRecordNotExist) {
-			encrypted, err := GeneratePasswordHash(InitAdminPassword)
-			if err != nil {
-				return err
-			}
-			if err := u.Store.Add(ctx, &model.User{
-				Name:      admin,
-				Alias:     model.DefaultAdminUserAlias,
-				Password:  encrypted,
-				UserRoles: []string{"admin"},
-			}); err != nil {
-				return err
-			}
-			// print default password of admin user in log
-			klog.Infof("initialized admin username and password: admin / %s", InitAdminPassword)
-		} else {
-			return err
+	}
+	if err := u.Store.Get(ctx, adminUser); err != nil {
```

Comment:
What happens if the user deletes the user with name `admin`?

Reply:
In the ideal situation only admin can do it. Then it become the initial state that the admin should be initialized.

Reply:
Fixed, judge the system status by checking if there is any user.